src/components/button/button-theme.scss

```diff
@@ -5,7 +5,7 @@ a.md-button.md-THEME_NAME-theme,
     &:hover {
       background-color: '{{background-500-0.2}}';
     }
-    &.md-focused {
+    &.md-focused:not(.md-no-focus) {
```

Comment:
NVM but i just feel like this is wrong, can't we apply `md-focused` only when `md-no-focus` is not available?